apps/aecore/lib/aecore/chain/block.ex

```diff
@@ -41,6 +58,26 @@ defmodule Aecore.Chain.Block do
     %Block{header: header, txs: []}
   end
 
+  def genesis_populated_trees do
```

Comment:
maybe we can move the genesis functions here and from governance in a specific genesis module, as they are only used once ever

Reply:
@thepiwo Done